app/partials/footer.html

```diff
@@ -1,3 +1,65 @@
 <div class="footer">
-  <p>♥ from the Yeoman team</p>
+  <div class="row f-menu">
+
+    <div class="col-md-2 col-md-offset-1">
+      <h3>Featured sale</h3>
+      <ul>
+        <li>Alexis Hudson</li>
```

Comment:
Also remove the dots in the list. There's a bootstrap class for this, or you can add the styles by your own